doc/changes/changes_6.6.2.md

```diff
@@ -0,0 +1,41 @@
+# Test Containers for Exasol on Docker 6.6.2, released 2023-09-25
+
+Code name: Fix vulnerability CVE-2023-42503
+
+## Summary
+
+This release fixes vulnerability CVE-2023-42503 in transitive dependency `org.apache.commons:commons-compress`.
+
+The release also updates the default Exasol version to 7.1.23
```

Comment:
If you need to run CI build again anyway, then please append a full-stop `.` after the sentence


```suggestion
The release also updates the default Exasol version to 7.1.23.
```